src/test/groovy/org/prebid/server/functional/AuctionSpec.groovy

```diff
@@ -23,4 +23,21 @@ class AuctionSpec extends BaseSpec {
         BidRequest.defaultBidRequest || "valid bid request"
         new BidRequest()             || "invalid bid request"
     }
+
+    def "PBS should increase request_time metric when auction was held"() {
```

Comment:
You're checking for wrong thing. This is the snippet from the documentation.
`request_time - timer tracking how long did it take for Prebid Server to serve a request`